packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -50,6 +50,10 @@ function moveBackendResourcesToCurrentCloudBackend(resources: $TSObject[]) {
   const amplifyCloudMetaFilePath = pathManager.getCurrentAmplifyMetaFilePath();
   const backendConfigFilePath = pathManager.getBackendConfigFilePath();
   const backendConfigCloudFilePath = pathManager.getCurrentBackendConfigFilePath();
+  const overridePackageJsonBackendFilePath = path.join(pathManager.getBackendDirPath(), 'package.json');
+  const overrideTsConfigJsonBackendFilePath = path.join(pathManager.getBackendDirPath(), 'tsconfig.json');
+  const overridePackageJsonCurrentCloudBackendFilePath = path.join(pathManager.getCurrentCloudBackendDirPath(), 'package.json');
```

Comment:
Do we need tsconfig.json copying anymore?

Reply:
I dont think we nneed to copy it as per our usecase now , but we can keep it handy to copy over tsconfig values from user to tsconfig-resource, when doing buildOverrideDir